tests/tests_pytorch/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -548,11 +548,11 @@ def test_step(self, batch, batch_idx):
         call(metrics={"train_loss": ANY, "epoch": 0}, step=0),
         call(metrics={"valid_loss_0_step": ANY, "valid_loss_2": ANY}, step=0),
         call(metrics={"valid_loss_0_step": ANY, "valid_loss_2": ANY}, step=1),
-        call(metrics={"valid_loss_0_epoch": ANY, "valid_loss_1": ANY, "epoch": 0}, step=0),
```

Comment:
The step value here corresponded to the training step. Now it's for the validation step